torch/__init__.py

```diff
@@ -48,8 +48,11 @@
         import torch._dl as _dl_flags
 
 old_flags = sys.getdlopenflags()
-sys.setdlopenflags(_dl_flags.RTLD_GLOBAL | _dl_flags.RTLD_NOW)
-
+sys.setdlopenflags(_dl_flags.RTLD_GLOBAL | _dl_flags.RTLD_LAZY)
+try:
+    import torch._nvrtc
```

Comment:
What's the difference between this and just always linking nvrtc into `_C` and opening it with `RTLD_LAZY`? Wouldn't that work too? It seems a lot simpler

Reply:
that does not work. it still tries to open libnvrtc.so immediately when it loads _C.so. (I tried it first.)